### PR TITLE
Section on automated testing issues

### LINKED ISSUE #49: Add note to readme on not using fake email addresses
When fake email addresses are used (i.e. for automated tests), it's incredibly easy to hit SES' limit of bounced emails and be blocked by the service.

I suggest we add a note to the readme to warn people of this.

### PR DESCRIPTION
**Pull request by shouston3**
Fixes #49 

Adding a section about the issues that may arise when running automated tests which send large volumes of emails.

**Note**: the example may be a bit long or wordy, perhaps we could move this out to a gist? Completely indifferent though.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,58 @@ non-technical people, focus on the fact that `.txt` email
...
+you can instead assume that the third party method will give back what they say they will give back
+(since these tests should already exist in the library you are using).
+
+Once you are sure there are [sufficient tests in place](https://codecov.io/github/dwyl/sendemail?branch=master) for the method you will stub, you may proceed with a test double.
+For this library there are significant benefits for using a test double for the `email` method (see #49).
+
+###### The following is a quick example of how to implement a test stub:
+If you have a helper function `notifyUser` which will subsequently call `require('sendemail').email`, this can be stubbed with:
+
+1. `npm install --D sinon`
```
Comment (nelsonic) [inline README.md]: @shouston3 prefer `--save-dev` for clarity, even though `--D` is equivalent. 😉

File: README.md [hunk 2]
```diff
@@ -212,6 +212,58 @@ non-technical people, focus on the fact that `.txt` email
...
 blocked by spam filters.
 
+### High volume of emails when running automated tests?
+When testing functions which will subsequently call methods in third party libraries,
+your tests no longer need to run through those methods,
+you can instead assume that the third party method will give back what they say they will give back
+(since these tests should already exist in the library you are using).
+
+Once you are sure there are [sufficient tests in place](https://codecov.io/github/dwyl/sendemail?branch=master) for the method you will stub, you may proceed with a test double.
+For this library there are significant benefits for using a test double for the `email` method (see #49).
```
Comment (iteles) [inline README.md]: We should probably link this to the issue itself for clarity but not a blocker. We can link and expand on this with the content of the issue at a later date so the issue can be closed 👍

File: README.md [hunk 3]
```diff
@@ -212,6 +212,58 @@ non-technical people, focus on the fact that `.txt` email
...
+
+###### The following is a quick example of how to implement a test stub:
+If you have a helper function `notifyUser` which will subsequently call `require('sendemail').email`, this can be stubbed with:
+
+1. `npm install --save-dev sinon`
```
Comment (iteles) [inline README.md]: 👍

**Comment by iteles** [comment]
Looking great, thanks so much for this @shouston3 !

**Pull request merged**